pkg/gadgets/trace/bind/tracer/tracer.go

```diff
@@ -276,13 +275,50 @@ func (t *Tracer) run() {
 			Options:   optionsToString(bpfEvent.Opts),
 			Interface: interfaceString,
 			Comm:      gadgets.FromCString(bpfEvent.Task[:]),
-			MountNsID: bpfEvent.MountNsId,
+			MountNsID: eventtypes.MountNsID{MountNsID: bpfEvent.MountNsId},
 		}
 
 		if t.enricher != nil {
-			t.enricher.EnrichByMntNs(&event.CommonData, event.MountNsID)
+			t.enricher.EnrichByMntNs(&event.CommonData, event.MountNsID.MountNsID)
```

Comment:
Would it be possible to change the name of `type MountNsID struct` to avoid clashing with the `MountNsID uint64` field. In that way we could just use `event.MountNsID` without having to specific the base type.

Reply:
Phew, absolutely. I wish I'd have thought about this a couple of hours ago. Well, I guess it's that not seeing the forest for the trees thing. Should we go back to naming it WithMntNsID / WithNetNsID?

/cc @blanquicet